components/develop/modules/ROOT/pages/deploy-a-website-on-skale.adoc

```diff
@@ -0,0 +1,84 @@
+= Deploying a website on SKALE Filestorage
+
+Follow these steps to deploy a decentralized website using SKALE Filestorage and the built in Web3 server.
+
+. Get a SKALE Chain Endpoint
++
+You can either use the xref:ima-sdk.adoc[SKALE-IMA-SDK] for a local development environment using Docker containers, or get access to a DappNet chain. You'll need to have the owner private key or a xref:skale-chain-access-control.adoc#_deployment_controller[deployment-authorized private key], to permissively upload files.
+
+. Prepare your files
++
+Assuming you already have built a statically generated website (if not checkout https://nextjs.org/, https://gohugo.io/, https://www.gatsbyjs.com/, or others, see a list here: https://jamstack.org/generators/). If you haven't already built a site, then here is a simple code example that you can copy and paste.
++
+.Save this as `index.html`
+[source, html]
+----
+<!DOCTYPE html>
+<link rel="stylesheet" type="text/css" href="styles.css">
+<html>
+    <body>
+        <h1>SKALE-ING</h1>
+        <p>My first decentralized website.</p>
+    </body>
+</html>
+----
++
+.Save this as `styles.css`
+[source, css]
+----
+body {
+    background-color: #ccc;
+}
+----
+
+. Upload to your SKALE Chain
++
+Assuming the `index.html` and `styles.css`, or static site files are stored in a folder called `test-skale`, and you have already performed `npm install @skalenetwork/filestorage.js~1.0.0`, then create a script file with the following code:
++
+.Save this as `filestorageUpload.js`.
+[source, javascript]
+----
+#! /usr/bin/env node
+
+const Filestorage = require('@skalenetwork/filestorage.js');
+const fs = require('fs');
+const Web3 = require('web3');
+
+// If not using the SDK, replace the endpoint below with your SKALE Chain endpoint
+let filestorage = new Filestorage('http://localhost:15000'); 
+
+// If not using the SDK, replace with the SKALE Chain owner key and address.
+let privateKey = process.env.TEST_PRIVATE_KEY || '<SKALE CHAIN OWNER KEY>';
+let address = process.env.TEST_ADDRESS || '<SKALE CHAIN OWNER ADDRESS>';
+
+let directoryPath = 'test-skale';
+
+// Bytes of filestorage space to allocate to an address
+const reservedSpace = 3 * 10 ** 8;
```

Comment:
Maybe better to write less mb in example or add that reservedSpace should be >= than sum of your uploaded files and dirs